specs/ServiceProtocol.md

```diff
@@ -67,15 +67,20 @@ Server will initiate a physical connection to Service, using WebSocket transport
 
 When a new client is connected to Service, a `OpenConnection` message will be sent by Service to Server.
 
+### Client migrated from another server
+
+When a client is migrated from another server, a `OpenConnection` message will be sent by Service to Server, with an `Asrs-Migrated-From` header given.
+
 ### Client Disconnect
 
 - When a client is disconnected from Service, a `CloseConnection` message will be sent by Service to Server.
 - When Server wants to disconnect a client, a `CloseConnection` message will be sent by Server to Service. Then Service will disconnect the phyical connection with the target client.
+	- In case of the `MigratableStatus` of the server connection is `Any`, the service will try to migrate the associated client to another server before dropping the connection.
```

Comment:
>In case of [](start = 3, length = 10)

Move to another section `### Client migrated`, also mention that  this one is transparent to clients but there might be message loss.